recipes/django-rest-hooks/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.2.2" %}
+
+package:
+  name: django-rest-hooks
+  version: {{ version }}
+
+source:
+  fn: django-rest-hooks-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/d/django-rest-hooks/django-rest-hooks-{{ version }}.tar.gz
+  sha256: c121ffbc1ee4a116bd9d5b39e3c97152f409d84dd4d13e121465d20a03ac0dcf
+
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
It seems that `django-rest-hooks` does not use `setuptools`. See https://github.com/zapier/django-rest-hooks/blob/master/setup.py#L1

You will need to drop the `--single-version-externally-managed --record=record.txt` here.